docs/application/native/guides/multimedia/media-playback.md

```diff
@@ -365,7 +361,7 @@ To initialize the player for use:
      }
      ```
 
-     The playback is interrupted and the callback triggered when, for example, another multimedia application is run. The player state automatically changes to `PLAYER_STATE_PAUSED` or `PLAYER_STATE_IDLE` according to the session policy. To get the current state, use the `player_get_state()` function.
+     The playback is interrupted and the callback triggered when, for example, another multimedia application is run. The player state automatically changes to `PLAYER_STATE_PAUSED` or `PLAYER_STATE_IDLE` according to the resource management policy. To get the current state, use the `player_get_state()` function.
```

Comment:
Suggest rephrasing, please verify whether my understanding is correct:

```suggestion
The playback is interrupted and the callback is triggered when, for example, some other multimedia application is running. The player state automatically changes to `PLAYER_STATE_PAUSED` or `PLAYER_STATE_IDLE` based on the resource management policy. To get the current state, use `player_get_state()`.
```

Reply:
**Original :** ... for example, another multimedia application is run.
**Suggested :** ... for example, some other multimedia application is running.

I think @eunhaechoi should check the meaning of each sentences as @surbhig12 asked.
To me, the original sentence seems to mean the other application gets started.
However the suggested sentence sounds like just checking the running state of the application.

So, would you please check if my understanding is correct?

Reply:
How about using " is run " as original expression? If player acquired H/W resources and it is not enough to share with other player, it will be interrupted regardless of player state. The resource is acquired to be PAUSED state and keep in PAUSED and PLAYING state.

Reply:
I also think keeping the original is better. @surbhig12 Would you please let us know your opinion?

Reply:
@eunhaechoi I think @surbhig12's suggestion is good. What's your opinion?

Reply:
@mijin85cho @surbhig12 I agree. thank you for your suggestion.